tracer/test/snapshots/Security.AspNetCore5AsmRemoteRules._.verified.txt

```diff
@@ -82,10 +82,6 @@
       runtime-id: Guid_1,
       span.kind: server,
       _dd.appsec.event_rules.version: 2.22.222,
-      _dd.appsec.fp.http.endpoint: http-get-7f4bf8ee-49fefa92-,
```

Comment:
Why has the finger printing been removed, when it's still working on other snapshots?

Reply:
As discussed this is okay, because the fingerprinting rules aren't in the custom ruleset used.